include/ada/mimesniff/util-inl.h

```diff
@@ -95,47 +95,46 @@ constexpr inline bool contains_only_http_quoted_string_tokens(
   return true;
 }
 
-inline std::string collect_http_quoted_string(std::string_view input,
-                                              size_t& position) {
+inline std::string collect_http_quoted_string(std::string_view& input) {
   std::string value{};
 
   // TODO: Assert: the code point at position within input is U+0022 (").
```

Comment:
I wouldn't have this TODO. Just document in the function that the caller is responsible for having checked that the string begins with `"`.